examples/typescript-reactflux/.gitignore

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
@arthurvr I did remove that line, but it's still showing up in the diff. Could be a `github` bug.